packages/@uppy/companion/test/__tests__/companion.js

```diff
@@ -136,14 +136,34 @@ describe('validate upload data', () => {
       .then((res) => expect(res.body.error).toBe('unsupported HTTP METHOD specified'))
   })
 
-  test('valid upload data is allowed', () => {
+  test('valid upload data is allowed - tus', () => {
     return request(authServer)
       .post('/drive/get/README.md')
       .set('uppy-auth-token', token)
       .set('Content-Type', 'application/json')
       .send({
         endpoint: 'http://master.tus.com/files',
         protocol: 'tus',
+        httpMethod: 'POST',
+        headers: {
+          customheader: 'header value'
+        },
+        metadata: {
+          mymetadata: 'matadata value'
+        },
+        fieldname: 'uploadField'
+      })
+      .expect(200)
+  })
+
+  test('valid upload data is allowed - s3-multipart', () => {
+    return request(authServer)
+      .post('/drive/get/README.md')
+      .set('uppy-auth-token', token)
+      .set('Content-Type', 'application/json')
+      .send({
+        endpoint: 'http://master.tus.com/files',
```

Comment:
Is that endpoint correct?

Reply:
it doesn't matter much in this case, it's only required to be a valid URL.

Reply:
Then maybe we should change it to something random, so that someone doesn't get confused accidentally?

Reply:
I have updated the URL to the correct tus URL